bukkit/src/main/java/de/maxhenkel/voicechat/config/ServerConfig.java

```diff
@@ -30,79 +30,79 @@ public ServerConfig(ConfigBuilder builder) {
                         "The port of the voice chat server",
-                        "Setting this to \"-1\" sets the port to the Minecraft servers port (Not recommended)"
+                        "Setting this to -1 sets the voice chat port to the Minecraft server port (Not recommended)"
                 );
         voiceChatBindAddress = builder
                 .stringEntry("bind_address", "",
-                        "The IP address to bind the voice chat server on",
-                        "Leave empty to use 'server-ip' of server.properties",
+                        "The IP address to bind the voice chat server to",
+                        "Leave blank to use the 'server-ip' property from server.properties",
                         "To bind to the wildcard address, use '*'"
                 );
         voiceChatDistance = builder
                 .doubleEntry("max_voice_distance", 48D, 1D, 1_000_000D,
-                        "The distance to where the voice can be heard"
+                        "The distance to which the voice can be heard"
                 );
         crouchDistanceMultiplier = builder
                 .doubleEntry("crouch_distance_multiplier", 1D, 0.01D, 1D,
-                        "The multiplier the voice distance will be reduced by when sneaking"
+                        "The multiplier of the voice distance when crouching"
                 );
         whisperDistanceMultiplier = builder
                 .doubleEntry("whisper_distance_multiplier", 0.5D, 0.01D, 1D,
-                        "The multiplier the voice distance will be reduced by when whispering"
+                        "The multiplier of the voice distance when whispering"
                 );
         voiceChatCodec = builder
                 .enumEntry("codec", Codec.VOIP,
-                        "The opus codec"
+                        "The Opus codec",
+                        "Valid values are 'VOIP', 'AUDIO', and 'RESTRICTED_LOWDELAY'"
                 );
         voiceChatMtuSize = builder
                 .integerEntry("mtu_size", 1024, 256, 10000,
-                        "The maximum size in bytes in a voice packet",
-                        "Set this to a lower value if your voice packets don't arrive"
+                        "The maximum size that voice packets are allowed to have (in bytes)"
                 );
         keepAlive = builder
                 .integerEntry("keep_alive", 1000, 1000, Integer.MAX_VALUE,
-                        "The frequency in which keep alive packets are sent",
+                        "The frequency at which keep-alive packets are sent (in milliseconds)",
                         "Setting this to a higher value may result in timeouts"
                 );
         groupsEnabled = builder
                 .booleanEntry("enable_groups", true,
-                        "If group chats are allowed"
+                        "Controls whether group chats are allowed"
```

Comment:
`Controls` sounds weird. Theres no need to rephrase this.

Reply:
I'm still undecided. As a programmer I see it as an if condition. If true then this option is active. Just "Controls whether" doesn't necessarily state what the option exactly does when enabled or disabled.

Reply:
In the good old days, when your voice chat mod still used the built-in Forge Config API (which, to be fair, wasn't that long ago, just early 2023 in voice chat mod version 2.3.x), as a user (or server owner), you could simply drop the [Configured mod](https://www.curseforge.com/minecraft/mc-mods/configured) into your "mods" directory, let it do its thing, and shortly after, you had a fully functional graphical configuration screen in the game, with *every single* option exposed, be it client, server, or even the so-called "common" ones (see below). No extra dependencies. No need to implement an external API. No additional work for the mod developer. Not a single extra line of code.

![Config-screen-generated-by-Configured-mod](https://github.com/user-attachments/assets/6c5b1468-feee-4f8c-b4cf-c4e0d66c0cab)

Why do I even bring this up?

First, I miss those days when I could easily and conveniently tweak *all* the settings for this mod directly through an in-game GUI, just like I can still do with your other mods such as Gravestone Mod and Corpse, which I've been using for many years now. And after granting myself the right permissions, I could even configure server-side settings directly from the client side, something that even NeoForge 21+ doesn't support (yet?).

Second, as you may have noticed from the screenshot above, the descriptions of what the various configuration options do shouldn't be overly technical but more user-friendly for casual players, because there are ways (or at least used to be) to expose these descriptions in the game by, for example, a modpack developer.

As a compromise, for example, I could modify only those option descriptions that are available through the Cloth Config configuration interface in the game to make them easier to translate. However, this would unfortunately reintroduce inconsistencies between the comments in the config files and the descriptions in the in-game tooltips, which is something I would rather avoid if possible. Moreover, if you decide in the future to expose more voice chat options through the in-game graphical interface (whether via the new config GUI in NeoForge 21+ or by some other means), it may once again raise the question of how to handle this.

Probably the best solution here would be to rephrase these descriptions so that there is no need for words like "if" or "whether" anymore. But this is not always possible. Even Mojang seemed to have a problem with this, as their descriptions sometimes start with "Controls ..." and at other times, they do not, see:
![Vanilla-tooltip-option-descriptions](https://github.com/user-attachments/assets/3b198d08-d870-4cf4-ad93-e2a45a3d4e9a)

**I am not here to impose anything on you, so if you think that X would be the best thing, I'll be happy to go along with it :)**

Reply:
I'm not a fan of the "Controls whether", if you have something else we could consider that, but for now I guess leave it like it was before.

Reply:
Okay, I've restored the original form both in the configuration file comments and in the in-game option descriptions in tooltips in the Cloth Config configuration interface. Is this what you wanted, or not exactly?